docs/easypoint.md

```diff
@@ -0,0 +1,25 @@
+# AS5013 (aka. 'EasyPoint')
+
+Module handles the AM5013 Two-dimensional magnetic position sensor with digital coordinates output
+
+Product page: https://ams.com/en/as5013
+
```

Comment:
Not a requirement, but a request for add in hardware like this would be an international link to buy these parts if any major vendors exist.